crates/astria-conductor/tests/blackbox/helpers/macros.rs

```diff
@@ -182,7 +182,8 @@ macro_rules! mount_update_commitment_state {
         mock_name: $mock_name:expr,
         firm: ( number: $firm_number:expr, hash: $firm_hash:expr, parent: $firm_parent:expr$(,)? ),
         soft: ( number: $soft_number:expr, hash: $soft_hash:expr, parent: $soft_parent:expr$(,)? ),
-        base_celestia_height: $base_celestia_height:expr
+        base_celestia_height: $base_celestia_height:expr,
+        expected_calls: $expected_calls:expr
```

Comment:
I had originally not done this since I had thought the default of having no `mock_name` or `expected_calls` fields would be good enough to cover both; just implemented another default for having `mock_name` but not `expected_calls` as well as you suggested :)